api/analyticsd/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/textileio/go-threads/util"
+	"github.com/textileio/textile/v2/api/analyticsd/service"
+	"github.com/textileio/textile/v2/cmd"
+)
+
+const daemonName = "analyticsd"
```

Comment:
Analytics now as its own service. This helped untangle some things and feels much better all around.